warehouse/ingest-core/src/main/java/datawave/ingest/config/RawRecordContainerImpl.java

```diff
@@ -437,6 +433,23 @@ public boolean equals(Object other) {
         return equals.isEquals();
     }
     
+    @Override
+    public int hashCode() {
+        int result = (int) (eventDate ^ (eventDate >>> 32));
+        result = 31 * result + (dataType != null ? dataType.hashCode() : 0);
+        result = 31 * result + (uid != null ? uid.hashCode() : 0);
+        result = 31 * result + (errors != null ? errors.hashCode() : 0);
+        result = 31 * result + (visibility != null ? visibility.hashCode() : 0);
+        result = 31 * result + (rawFileName != null ? rawFileName.hashCode() : 0);
+        result = 31 * result + (int) (rawFileTimeStamp ^ (rawFileTimeStamp >>> 32));
+        result = 31 * result + (int) (rawRecordNumber ^ (rawRecordNumber >>> 32));
+        result = 31 * result + (ids != null ? ids.hashCode() : 0);
+        result = 31 * result + Arrays.hashCode(rawData);
+        result = 31 * result + (auxData != null ? auxData.hashCode() : 0);
+        result = 31 * result + (securityMarkings != null ? securityMarkings.hashCode() : 0);
+        return result;
```

Comment:
Looks like this hashCode method doesn't include auxMap whereas equals compares it.

Reply:
Thanks. Fixed in next commit